extensions/vscode/src/views/homeView.ts

```diff
@@ -1034,8 +1053,13 @@ export class HomeViewProvider implements WebviewViewProvider, Disposable {
         },
       ).finally(() => Disposable.from(...toDispose).dispose());
 
+      // If user selected create new, then switch over to that flow
+      if (deployment?.label === createNewDeploymentLabel) {
+        return this.showNewDeploymentMultiStep(Views.HomeView);
+      }
+
```

Comment:
Easiest way to handle creating a new deployment from within the selection quickpick.